docs/podman-run.1.md

```diff
@@ -809,18 +811,21 @@ Set the UTS mode for the container
 
 **NOTE**: the host mode gives the container access to changing the host's hostname and is therefore considered insecure.
 
-**--volume**, **-v**[=*[HOST-DIR:CONTAINER-DIR[:OPTIONS]]*]
+**--volume**, **-v**[=*[HOST-DIR-OR-VOUME-NAME:CONTAINER-DIR[:OPTIONS]]*]
```

Comment:
Please add an example showing how the volume-name is used down below in the examples

Reply:
> Please add an example showing how the volume-name is used down below in the examples

I've already add an example a paragraph after. Maybe it should be more clear?

Reply:
It was just that you added this "or volume name" thought to the docs, but I didn't see any additional examples.  Maybe in the example below add a little verbiage and a second example?

To add a volume using a host directory:

{example}

To add a volume using the volume name:

{example}